input/docs/getting-started/compelling-example.md

```diff
@@ -0,0 +1,467 @@
+Title: A Compelling Example
+Order: 10
+---
+
+Let's create a simple application demonstrating a number of ReactiveUI functionalities, without getting into too many under-the-hood details. We will create a WPF application, which will allow us to search through NuGet public repositories. The full code of the application is shown at the end of this chapter, and we will show relevant snippets as we go.
+
+### 1. Create the project
+
+In <a href="[https://visualstudio.microsoft.com/](https://visualstudio.microsoft.com/)">Visual Studio</a>, create a new WPF application (.NET 4.6.1 or above), use `ReactiveDemo` app name. Our view has been already created for us, the `MainWindow`, so we will proceed with creating our ViewModel.
```

Comment:
Probably should update this to .net core 3.1 now

Reply:
Haha, things are out of date too quickly! 😅